sql/attribute_validator_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 The SQLFlow Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sql
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestValidateAttribute(t *testing.T) {
+	a := assert.New(t)
+	attrs := make(map[string]*attribute)
+	attrs["train.steps"] = &attribute{FullName: "train.steps", Name: "steps", Prefix: "train", Value: "100"}
```

Comment:
I saw some cases also using `WITH BATCHSIZE=2` instead of `WITH train.batchsize` like: 
https://github.com/sql-machine-learning/sqlflow/blob/77cac234c571ec0af6ddd1d30a1d5c54e28f674e/cmd/sqlflowserver/main_test.go#L616-L620

maybe we need to make the syntax to be consistent and refine the example code?

Reply:
You're right, will update `codegen.go` to use `expression_resolver.go` so that all attributes are of the same format.